exchanges/bithumb/bithumb_websocket.go

```diff
@@ -28,9 +28,7 @@ const (
 	tradeTimeLayout  = time.DateTime + ".000000"
 )
 
-var (
-	location *time.Location
-)
+var location *time.Location
```

Comment:
OOS: Localise this to Bithumb struct and set in setdefaults or set it globally straight away at runtime so it doesn't race when you make multiple instances of bithumb and call setup

```go
var location = func() *time.Location {
	tz, err := time.LoadLocation("Asia/Seoul")
	if err != nil {
		panic(err)
	}
	return tz
}()
```